sqlglot/dialects/spark2.py

```diff
@@ -199,6 +221,14 @@ class Generator(Hive.Generator):
             exp.Reduce: rename_func("AGGREGATE"),
             exp.StrToDate: _str_to_date,
             exp.StrToTime: lambda self, e: f"TO_TIMESTAMP({self.sql(e, 'this')}, {self.format_time(e)})",
+            exp.Select: transforms.preprocess(
+                [
```

Comment:
make the base hive transform an array so that you can call them here so you're not copy and pasting